critiquebrainz/frontend/templates/search/selector_results.html

```diff
@@ -131,6 +131,34 @@
     </td>
     <td>
       <input type="radio" name="label" value="{{ result['id'] }}">
+      <input type="radio" name="artist" value="{{ result['id'] }}">
```

Comment:
I opened CB-375 about this
How much do these results differ from `search/results.html`? Are we able to merge them into a single file?

Reply:
I think the files are mostly similar and we can merge them together.